table/tables/tables.go

```diff
@@ -916,6 +916,11 @@ func GetColDefaultValue(ctx sessionctx.Context, col *table.Column, defaultVals [
 
 // AllocAutoID implements table.Table AllocAutoID interface.
 func (t *tableCommon) AllocAutoID(ctx sessionctx.Context) (int64, error) {
```

Comment:
Maybe `AllocAutoIncrementValue` is more accurate.

Reply:
addressed, thanks! @coocood